src/wp-admin/setup-config.php

```diff
@@ -416,16 +416,16 @@ function setup_config_display_header( $body_classes = array() ) {
 <textarea id="wp-config" cols="98" rows="15" class="code" readonly="readonly"><?php echo $config_text; ?></textarea>
 <p><?php _e( 'After you&#8217;ve done that, click &#8220;Run the installation&#8221;.' ); ?></p>
 <p class="step"><a href="<?php echo $install; ?>" class="button button-large"><?php _e( 'Run the installation' ); ?></a></p>
-<script>
-(function(){
-if ( ! /iPad|iPod|iPhone/.test( navigator.userAgent ) ) {
-	var el = document.getElementById('wp-config');
-	el.focus();
-	el.select();
-}
-})();
-</script>
 			<?php
+			$js = <<<'JS'
```

Comment:
Forgot to print `$js` I think?

Looks like this could be moved to `setup-config.js` anyway though

Reply:
In this commit I moved both to setup-config: https://github.com/WordPress/wordpress-develop/pull/551/commits/ee8afc76faec52eda0107d814238f4f4e5b9d1c9
Then I moved back this part because my host said it's orthogonal to the effort; when I moved it back I forgot to print it.

Reply:
Will fix soon.